lib/new-admin/admin-server.js

```diff
@@ -48,6 +48,7 @@ app.use(express.static(path.resolve(__dirname, '..', '..', 'public')))
 app.use(cleanUserSessions(USER_SESSIONS_CLEAR_INTERVAL))
 app.use(computeSchema)
 app.use(findOperatorId)
+app.use(loadSanctionLists)
```

Comment:
Wouldn't it be easier (maybe better) to run `load()` here, instead of erroring out?

https://github.com/lamassu/lamassu-server/blob/d446b0635af87ab7570cf9c84ef574709f4c04e5/lib/ofac/index.js#L48-L51

No need to run the `loadSanctionLists()` on every request. Only the first `match()` would be slower.